src/app/design/frontend/base/default/template/payment/form/omise/omiseoffsiteinstallment.phtml

```diff
@@ -14,11 +22,18 @@ $backends = $this->getInstallmentBackends();
                 <div class="logo-<?=$backend->_bankcode;?>"></div>
             </div>
             <div class="omise-banking-text-wrapper">
-                <span class="title"><?=$backend->_bankname;?></span><br/>
+                <span class="title"><?=$installmentText;?></span><br/>
                 <select name="payment[terms_<?=$backend->_id;?>]" id="payment_term_<?=$backend->_id;?>">
-                    <?php foreach ($backend->allowed_installment_terms as $termMonths) { ?>
-                        <option value="<?=$termMonths;?>"><?=$termMonths;?> <?=$this->__('months');?></option>
-                    <?php } ?>
+
+                    <?php foreach ($backend->allowed_installment_terms as $termMonths) {
+                        $monthlyPaymentAmount = $this->getMonthlyPaymentAmount($saleAmount, $termMonths, $backend->_interest_rate);
+                        // This will need to change if we introduce instalment payments for currencies with different subunit settings
+                        if ($monthlyPaymentAmount >= ($backend->_monthly_minimum_subunits_thb/100)) {
+                            $monthlyPaymentText = $currencyHelper->toCurrency($monthlyPaymentAmount) . ' ' .$this->__('/ month'); ?>
+                            <option value="<?=$termMonths;?>"><?=$termMonths;?> <?=$this->__('months');?> - <?=$monthlyPaymentText;?></option>
+                        <?php } 
```

Comment:
I would format that code adding more spaces and make it more readable.

Reply:
i saw that code big bigger when was making review.. there was some comment above as well... keep in mind that my comment above is to bigger block that visible here.